jaba/src/pages/AdminLoginPage/LoginSignInPage.module.css

```diff
@@ -0,0 +1,163 @@
+.splitScreen {
+    height: calc(100vh);
+    display: flex;
+    flex-direction: row;   
+}
+
+.leftSide {    
+    width: 50%;    
+    display: flex;    
+    align-items: center;
+    justify-content: center;
+    background-color: #023C47;
+}
+
+.loginRectangle {
+    width: 505px;
+    height: 701px;
+    top: 216px;
+    left: 125px;
+    border-radius: 10px;
+    border: 0.5px;
+    background: linear-gradient(0deg, #FFFFFF, #FFFFFF);
```

Comment:
Can't you just do "background: white" here?

Reply:
done